mkosi/__init__.py

```diff
@@ -3560,7 +3570,7 @@ def insert_generated_root(
     assert image is not None
 
     with complete_step("Inserting generated root partition"):
-        args.root_size = insert_partition(
+        insert_partition(
             args,
```

Comment:
From what I can see, the assigned value here is never read again so I just removed it. Would be great to have a second look here though, I might have missed something.

Reply:
I don't think it is used afterwards, but I see no reason not to keep it around, since I don't think it's a maintenance burden. Maybe an explanatory comment would be better.